### PR TITLE
docs: foundation-forms with live examples PTL-1516

### PR DESCRIPTION
**Pull request by SzymonZur**
ATENTION! YOU SHOULD NOW BRANCH FROM PREPROD WHEN YOU UPDATE 

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you done a trial build to check all new or changed links?

Is there anything else you would like us to know?

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

https://github.com/user-attachments/assets/d696c2c9-a11f-41bb-8266-4fe0095c7729

### CODE REVIEW

File: docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+
+## Examples 
+
+### Server-side JSON schema
+
+Here is an example of a simple table defined in an application's **-tables-dictionary.kts** file. It is a simple table with three fields:
+
+**smart-forms-examples-tables-dictionary.kts**
+```kts
+ tables {
```
Comment (patrickoneill-genesis) [inline docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx:38]: Can you move this to it's own file and import it (as discussed on calls)?

File: docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <foundation-form
+      design-system-prefix="rapid"
+      resourceName="EVENT_TRADE_INSERT"
+    ></foundation-form>
+  `,
+  styleUrls: ['./my.component.css'],
```
Comment (patrickoneill-genesis) [inline docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx:152]: if we don't have the css file can we just remove it?

**Review by patrickoneill-genesis — COMMENTED** [review]
Looks good, only thing is the backend part can that be moved to it's own file for re use elsewhere - say on an entity management page?

File: docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+    }
+    ...
+}
+...
+```
+
+To insert the automatically-generated JSON schema for this resource in your **template** file, use the code below:
+
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
```
Comment (patrickoneill-genesis) [inline docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx:79]: I would like @submit-success and @submit-failure events added here?

When I re wrote the forms docs I didn't event know they existed - happy for it to be spun out to a separate PR

Comment (SzymonZur) [reply docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx:79]: Why here? All events are included and described in the DOM API section.

Comment (patrickoneill-genesis) [reply docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx:79]: Just to see an example of them in action.

**Review by skawian — APPROVED** [review]

File: docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx (hunk 2 above)
Comment (SzymonZur) [reply docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx]: [36a4841](https://github.com/genesiscommunitysuccess/docs/pull/1872/commits/36a4841084273021e30e1d3b10e5dd88d6a271bc)

File: docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx (hunk 3 above)
Comment (SzymonZur) [reply docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx:79]: I can add it in separate PR

**Review by patrickoneill-genesis — APPROVED** [review]

**Review by wjhendry — APPROVED** [review]
Thanks.

**Pull request merged**